.github/CHANGELOG.md

```diff
@@ -1,15 +1,4 @@
-# Release 0.9.0-dev (development release)
-
-<h3>New features since last release</h3>
-
-* Added the `qml.qnodes.PassthruQNode` class for simulated QNodes that appear as white boxes
-  to an external autodifferentiation (AD) framework, and hence can be directly differentiated
-  by it. Note that the simulator device executing the PassthruQNode has to be compatible with
-  the external AD framework. Currently the only such device supported by PennyLane is
-  `default.tensor.tf`, compatible with TensorFlow 2.
-  [(#488)](https://github.com/XanaduAI/pennylane/pull/488)
-
-<h3>Breaking changes</h3>
```

Comment:
How come these were deleted? :)

Reply:
The plan is to advertise and polish this feature for v0.9 🙂 Currently, the PassthruQNode is not available through the decorator, so it's not userfacing